script-http-request/catalog.json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
I've tested the command and it was imported and executed correctly and I was able to read my result in the output. Just one comment: was not sure shall we provide some expected behaviour or use case within the sample so that the user can play with the command following some guidance / expected result to be reached. 
However, perhaps that can be created as a tutorial by @bisinho and added as a link within the sample. Nevertheless, just adding some expected behavior here might be quite useful , i.e. 

GET:  [endpoint in SAP] 
Output transformation [results]

Reply:
Agree. I've added concrete input with screenshots